src/components/ExpensifyText.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import _ from 'underscore';
+
+// importing react-native Text as base for ExpensifyText component
```

Comment:
@MonilBhavsar @madmax330 I was a bit on the fence about whether this comment was necessary in this scenario. It feels a bit obvious to me why we're ignoring the rule in this case and by adding this comment we end up having to add the blank line above which feels kinda ugly. What do y'all think?

Reply:
Yes, we can remove the comment as it doesn't explain much